packages/utils/src/path.ts

```diff
@@ -194,7 +194,7 @@ export const path: Path = {
 
         if (isFile || isHttp || isWindows)
         {
-            const arr = isFile?.[0] || isHttp?.[0] || isWindows?.[0];
+            const arr = isFile?.[0] || isHttp?.[0] || isWindows?.[0] || '';
```

Comment:
```suggestion
            const arr = (isFile?.[0] || isHttp?.[0] || isWindows?.[0]) as string;
```

Since it won't be `undefined`.